### PR TITLE
Update features.rst

### PR DESCRIPTION
**Pull request by namloc2001**
Adding further detail regarding JIRA integration.

### CODE REVIEW

File: docs/features.rst [hunk 1]
```diff
@@ -776,6 +776,14 @@ JIRA Integration
 
 DefectDojo's JIRA integration is bidirectional. You may push findings to JIRA and share comments. If an issue is closed in JIRA it will automatically be closed in Dojo.
 
+**NOTE:** These steps will configure the necessary webhook in JIRA and add JIRA integration into DefectDojo. This isn't sufficient by itelf, you will need to configure products and findings to push to JIRA. On a product's settings page you will need to define a:
+ - Project Key (and this project must exist in JIRA)
+ - JIRA Configuration (select the JIRA configuration that you create in the steps below)
+ - Component (can be left blank)
+
+Then elect (via tickbox) whether you want to 'Push all images', 'Enable engagement epic mapping' and/or 'Push notes'. Then click on 'Submit'.
```
Comment (valentijnscholten) [inline docs/features.rst]: some typos I think it's  "push all findings"

**Comment by valentijnscholten** [comment]
thanks for the PR

File: docs/features.rst (hunk 1 above)
Comment (namloc2001) [reply docs/features.rst]: Good spot. It's "push all issues". Have updated.

**Comment by valentijnscholten** [comment]
thanks for contributing

**Pull request merged**